CHANGES.rst

```diff
@@ -5,7 +5,7 @@ Changes to be released in next version
  * 
 
 🙌 Improvements
- * 
+ * SDK: Expose the HTTP response in MXHTTPOperation (vector-im/element-ios/issues/4206).
```

Comment:
```suggestion
 * MXHTTPOperation: Expose the HTTP response (vector-im/element-ios/issues/4206).
```

It is hard to update the readme in one line. We try to sort it in categories as much as possible. Category can be a feature or just the class itself for a tool.
Note that `SDK` is meaningless here. We are in the MatrixSDK repo.